cpp/velox/compute/VeloxBackend.cc

```diff
@@ -60,6 +60,12 @@ const std::string kEnableUserExceptionStacktrace =
     "spark.gluten.sql.columnar.backend.velox.enableUserExceptionStacktrace";
 const std::string kEnableUserExceptionStacktraceDefault = "true";
 
+const std::string kGlogVerboseLevel = "spark.gluten.sql.columnar.backend.velox.glogVerboseLevel";
+const std::string kGlogVerboseLevelDefault = "0";
+
+const std::string kGlogSeverityLevel = "spark.gluten.sql.columnar.backend.velox.glogMinLogLevel";
```

Comment:
Should the key text be `spark.gluten.sql.columnar.backend.velox.glogSeverityLevel` ?

Reply:
Yes, my mistake.